drivers/serial/uart_ns16550.c

```diff
@@ -1243,6 +1318,14 @@ static const struct uart_driver_api uart_ns16550_driver_api = {
 		irq_enable(DT_INST_IRQN(n));                                  \
 	}
 
+#define UART_NS16550_IRQ_CONFIG_ACPI1(n)                                                           \
+	static void irq_config_func##n(const struct device *dev)                                   \
+	{                                                                                          \
+	}
```

Comment:
Why is this empty for ACPI1?

Reply:
For instances on ACPI node, both IRQ and MMIO config as part of init routine. So ACPI irq config is add as stub function.

Reply:
updated the code